functions-methods.js

```diff
@@ -3,31 +3,88 @@
 
 
 /* Opdracht  1 */
+
 // Schrijf een functie genaamd getEmailDomain, die een emailadres verwacht en de domeinnaam teruggeeft. Een domeinnaam is hetgeen dat na het @ in het adres staat
 // ---- Verwachte uitkomsten:
 // getEmailDomain("n.eeken@novi-education.nl") geeft novi-education.nl
+
+function getEmailDomain(email) {
+    const parts = email.split('@');
+    return parts[1];
+}
+console.log (getEmailDomain("n.eeken@novi-education.nl"));
+
 // getEmailDomain("t.mellink@novi.nl") geeft novi.nl
-// getEmailDomain("a.wiersma@outlook.com") geeft outlook.com
 
+function getEmailDomain(email) {
+    const parts = email.split('@');
+    return parts[1];
+}
+console.log (getEmailDomain("t.mellink@novi.nl"));
 
+// getEmailDomain("a.wiersma@outlook.com") geeft outlook.com
+
+function getEmailDomain(email) {
+    const parts = email.split('@');
+    return parts[1];
+}
+console.log (getEmailDomain("a.wiersma@outlook.com"));
 
 
 /* Opdracht  2 */
 // Schrijf een functie genaamd typeOfEmail, die een emailadres verwacht. De functie checkt of het emailadres een novi domein heeft (medewerker), een novi-education domein (student), of extern domein (zoals gmail of outlook)
 // ---- Verwachte uitkomsten:
+
 // typeOfEmail("n.eeken@novi-education.nl") geeft "Student"
+
+    function typeOfEmail(email) {
+        const parts = email.split('@');
+        const domain = parts[1];
```

Comment:
Hier is het gebruik van een variabele wel correct aangezien je deze vaker gebruikt binnen de functie.